src/test/java/com/wikia/webdriver/elements/oasis/components/comment/ArticleComment.java

```diff
@@ -4,6 +4,7 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
 import com.gargoylesoftware.htmlunit.ElementNotFoundException;
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.WebDriver'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)